packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.slow-spec.ts

```diff
@@ -733,6 +733,7 @@ describe('ripgrep-search-in-workspace-server', function () {
         expect(errorString).contains('could not find the ripgrep (rg) binary');
     });
 
+    /*
```

Comment:
There is a race condition here where `spawn()` returns an uninitialised process.